examples/AspNetCore/Controllers/WeatherForecastController.cs

```diff
@@ -24,16 +26,22 @@ public class WeatherForecastController : ControllerBase
 {
     private static readonly string[] Summaries = new[]
     {
-            "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching",
+        "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching",
     };
 
     private static readonly HttpClient HttpClient = new();
+    private static readonly ActivitySource MyActivitySource = new ActivitySource("Examples.AspNetCore");
+    private static readonly Meter MyMeter = new Meter("Examples.AspNetCore");
 
     private readonly ILogger<WeatherForecastController> logger;
+    private readonly Counter<int> counter;
```

Comment:
Curious - why isn't this static? 👀

Reply:
This is the console demo https://github.com/open-telemetry/opentelemetry-dotnet/blob/c0f927f9a36ec76a76d18d7f3c1a5350dc42d316/docs/metrics/getting-started/Program.cs#L26.